src/build/buildController.ts

```diff
@@ -35,59 +36,62 @@ export class BuildController {
         this.buildExecutor = new BuildExecutor(context, platformInformation, environmentController, eventStream);
     }
 
-    public async build(uri: vscode.Uri, credential: Credential): Promise<void> {
-        if (!this.trySetAvailableFlag()) {
+    public async build(correlationId: string, uri: vscode.Uri, credential: Credential): Promise<void> {

```

Comment:
`async` is required to use `await` inside the current function, since there is just oneline in the caller, so we don't need to await

---
In reply to: [380557413](https://github.com/docascode/vscode-docs-build/pull/44#discussion_r380557413) [](ancestors = 380557413)